packages/forma-36-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,158 @@
+
+### Position (`position`) <a name="position" />
+
+```jsx
+function ModalExample() {
+  const [isShown, setShown] = React.useState(false);
+  const [position, setPosition] = React.useState('top');
+  return (
+    <React.Fragment>
+      <Form>
+        <FieldGroup row>
+          <Button
+            buttonType="positive"
+            onClick={() => {
+              setPosition('top');
+              setShown(true);
+            }}
+          >
+            Show top modal
+          </Button>
+          <Button
+            onClick={() => {
+              setPosition('center');
+              setShown(true);
+            }}
+          >
+            Show center modal
+          </Button>
+        </FieldGroup>
+      </Form>
+      <Modal title="Centered modal" isShown={isShown} position={position}>
+        {() => (
+          <React.Fragment>
+            <Modal.Header title="Title" />
+            <Modal.Content>Hello from controlled modal window</Modal.Content>
+            <Modal.Controls>
+              <Button buttonType="positive" onClick={() => setShown(false)}>
+                Confirm
+              </Button>
+              <Button buttonType="muted" onClick={() => setShown(false)}>
+                Close
+              </Button>
+            </Modal.Controls>
+          </React.Fragment>
+        )}
+      </Modal>
+    </React.Fragment>
+  );
+}
+```
+
+### Sizes (`size`)
+
+```jsx
+function ModalExample() {
+  const [isShown, setShown] = React.useState(false);
+  const [size, setSize] = React.useState('medium');
+  return (
+    <React.Fragment>
+      <Form>
+        <SelectField
+          onChange={e => {
+            setSize(e.target.value);
+          }}
+          labelText="Select a width"
+        >
+          <Option value="small">small</Option>
+          <Option value="medium">medium</Option>
+          <Option value="large">large</Option>
+          <Option value="fullWidth">full width</Option>
+          <Option value="200px">200px (specific width value)</Option>
+        </SelectField>
+        <Button
+          onClick={() => {
+            setShown(true);
+          }}
+        >
+          Show modal
+        </Button>
+      </Form>
+      <Modal title="Centered modal" isShown={isShown} size={size}>
+        {() => (
+          <React.Fragment>
+            <Modal.Header title="Title" />
+            <Modal.Content>Hello from controlled modal window</Modal.Content>
+            <Modal.Controls>
+              <Button buttonType="positive" onClick={() => setShown(false)}>
+                Confirm
+              </Button>
+              <Button buttonType="muted" onClick={() => setShown(false)}>
+                Close
+              </Button>
+            </Modal.Controls>
+          </React.Fragment>
+        )}
+      </Modal>
+    </React.Fragment>
+  );
+}
+```
+
+## Copy considerations <a name="copy-considerations" />
+
+TODO
```

Comment:
- Practice progressive disclosure. Eliminate extra details to keep copy concise and actionable
- Focus on the objective of the modal dialog. Start each sentence with the objective when possible
- To enable users to navigate quickly, write concise sentences that are easy to understand at a glance 
- Limit the number of concepts in each sentence. Write simple sentences for all audiences
- To keep UX copy actionable use present tense for user interactions
- Avoid “never,” “always” and other absolutes
- Use contractions when possible
- Avoid exclamation marks